editors/sc-ide/widgets/audio_status_box.cpp

```diff
@@ -87,19 +89,39 @@ AudioStatusBox::AudioStatusBox(ScServer* server, QWidget* parent): StatusBox(par
     connect(this, &AudioStatusBox::decreaseVolume, [=]() { server->changeVolume(-0.5); });
 
     connect(this, &AudioStatusBox::increaseVolume, [=]() { server->changeVolume(+0.5); });
+
+    auto const main = Main::instance();
+    applySettings(main->settings());
+    connect(main, &Main::applySettingsRequest, this, &AudioStatusBox::applySettings);
 }
 
+void AudioStatusBox::applySettings(Settings::Manager* settings) {
+    auto backgroundColor = settings->getThemeVal("text").background().color();
+
+    mStatisticsLabel->setBackground(backgroundColor);
+    mVolumeLabel->setBackground(backgroundColor);
+    mMuteLabel->setBackground(backgroundColor);
+    mRecordLabel->setBackground(backgroundColor);
+
+    // used if e.g. server is not recording is not recording
+    noActionColor = Qt::black;
+
+    unresponsiveColor = settings->getThemeVal("number").foreground().color();
+    runningColor = settings->getThemeVal("symbol").foreground().color();
+    notRunningColor = settings->getThemeVal("text").foreground().color();
+    warningColor = settings->getThemeVal("postwindowerror").foreground().color();
```

Comment:
totally cosmetic - if we're using the "error" color here, maybe the variable would be better called `errorColor`?